### PR TITLE
feat: init model group page

### PR DESCRIPTION
**Pull request by ruanyl**
+ Navigate to model group page after model creation succeed
+ Added a new route `/model-registry/model/:id` for model group

### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/hooks/tests/use_model.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      setTimeout(() => reject(new Error()), 1000);
+    });
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should return the target model', async () => {
+    jest.useFakeTimers();
```
Comment (wanglam) [inline public/hooks/tests/use_model.test.ts:34]: What about move to beforeEach and afterEach?

File: public/hooks/use_model.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  });
+  return data[0];
+};
+
+export const useModel = (modelId: string) => {
+  const [loading, setLoading] = useState(false);
+  const [data, setData] = useState<ModelSearchItem>();
+  const [error, setError] = useState<Error>();
+
+  useEffect(() => {
```
Comment (wanglam) [inline public/hooks/use_model.ts:25]: Could we useFetcher here?

File: public/hooks/tests/use_model.test.ts (hunk 1 above)
Comment (ruanyl) [reply public/hooks/tests/use_model.test.ts:34]: Sounds good

File: public/hooks/use_model.ts (hunk 2 above)
Comment (ruanyl) [reply public/hooks/use_model.ts:25]: Good point! Totally forgot about it

File: public/components/register_model/register_model.tsx [hunk 3]
```diff
@@ -86,7 +86,10 @@ export const RegisterModelForm = () => {
   const onSubmit = useCallback(
     async (data: ModelFileFormData | ModelUrlFormData) => {
       try {
-        const onComplete = () => {
+        const onComplete = (modelId: string) => {
```
[the remark this answers is not in the document]
Comment (wanglam) [reply public/components/register_model/register_model.tsx:89]: Sure. I think it won't block the PR if take a long time to get the final UX.

**Review by wanglam — APPROVED** [review]

**Pull request merged**